sdk/compatibility/update-versions.sh

```diff
@@ -5,7 +5,7 @@
 set -euo pipefail
 
 cd "$(dirname "$0")"
-eval "$(../dev-env/bin/dade-assist)"
+eval "$(cd ..; dev-env/bin/dade-assist)"
```

Comment:
`dade-assist` does not actually work that well when invoked from a non-root cwd. Specifically, this causes `semver` to not be found, though it looks like we no longer need it in the rest of this script.